pom.xml

```diff
@@ -72,155 +68,284 @@
 
 		<module>jzy3d-tutorials</module>
 
-
 		<!-- Maths -->
 		<module>jzy3d-jdt-core</module>
 		<module>jzy3d-svm-mapper</module>
 		<module>jzy3d-tools-libsvm</module>
 
-
 		<module>jzy3d-io-xls</module>
 		<module>jzy3d-io-reports</module>
+
 	</modules>
 
+	<scm>
+		<connection>scm:git:https://github.com/jzy3d/jzy3d-api</connection>
+	</scm>
+	<issueManagement>
+		<system>Github</system>
+		<url>https://github.com/jzy3d/jzy3d-api/issues</url>
+	</issueManagement>
+
+	<!-- Used to deploy to Jzy3d repository -->
+	<distributionManagement>
+		<repository>
+			<id>jzy3d-ftp</id>
+			<name>Jzy3d Maven Folder</name>
+			<url>ftp://ftp.cluster013.ovh.net/maven/releases</url>
+		</repository>
+		<snapshotRepository>
+			<id>jzy3d-ftp</id>
+			<name>Jzy3d Maven Folder SNAPSHOTS</name>
+			<url>ftp://ftp.cluster013.ovh.net/maven/snapshots</url>
+		</snapshotRepository>
+	</distributionManagement>
+
+	<!--<distributionManagement> <snapshotRepository> <id>ossrh</id> <url>https://oss.sonatype.org/content/repositories/snapshots</url> 
+		</snapshotRepository> <repository> <id>ossrh</id> <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url> 
+		</repository> </distributionManagement> -->
+
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<version.jzy3d>${project.version}</version.jzy3d>
-		<version.java.source>1.8</version.java.source>
-		<version.java.target>1.8</version.java.target>
-		<version.mvn.compiler>3.0</version.mvn.compiler>
-		<version.mvn.ftp>2.10</version.mvn.ftp>
-		<version.mvn.deploy>2.4</version.mvn.deploy>
-		<version.mvn.javadoc>2.9.1</version.mvn.javadoc>
-		<version.mvn.release>2.5.3</version.mvn.release>
+		<version.java>1.8</version.java>
+		<version.swt>3.105.3</version.swt>
```

Comment:
I am surprised by this : why did you keep SWT as a configurable dependency while all other have defaults in main pom file?

Reply:
This is because the handling of SWT dependencies is not straight forward because Eclipse-p2's dependency-handling and maven's dependency-handling is mixed. Therefore you have to specify a dependency to a specific platform (linux, windows, mac) to compile successfully. If you only specify the dependency org.eclipse.platform::org.eclipse.swt you only get an empty hull with nothing in it. So in order to compile the sources of jzy3d successfully but without forcing projects that depend on jzy3d's SWT native project to depend on all SWT platforms jzy3d does I specified them as optional dependency. 

I wrote some explanation about it into the `jzy3d-native-jogl-swt/pom.xml`.
And this requires multiple locations were the SWT version is required. Therefore I created a parameter, so they don't have to be updated all in case another SWT version is used one day.